db/migrations/002_cla_url_text.up.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+ALTER TABLE signatures
+    ADD COLUMN ClaTextUrl VARCHAR(250) NOT NULL,
```

Comment:
Cool! Will DB need some default values for any existing records for these? (or just delete 'em)?

Reply:
Good point!